packages/contracts/deploy/11_demo.js

```diff
@@ -0,0 +1,15 @@
+const { ethers } = require("hardhat");  
```

Comment:
Shouldn't this modify the existing deployment to customise it for different networks? We don't need two deployments deploying the same thing.

Reply:
It would make more sense, then to make 10_offchain_resolver just deploy the resolver, and add a new deployment that sets subnodes etc and only runs on the hardhat network. We definitely shouldn't have two separate deployments that both deploy the same contract.